serverless.yml

```diff
@@ -0,0 +1,35 @@
+#serverless.yml
```

Comment:
# note to reviewers
this repo is public (a fork, not a private one) so keep it generic